ccm_web/client/src/pages/FormatThirdPartyGradebook.tsx

```diff
@@ -132,12 +133,6 @@ export default function FormatThirdPartyGradebook (props: FormatThirdPartyGradeb
     )
   }
 
-  useEffect(() => {
-    if (file !== undefined) {
-      parseFile(file)
-    }
-  }, [file])
-
```

Comment:
This isn't related to the issue, but this hook isn't really needed, and it's something I've factored out in other pages/components. We can just use the `onUploadComplete` to set the file in state and call `parseFile`.

Reply:
OK, I test this and things are working as expected.